CONTRIBUTING.md

```diff
@@ -56,19 +72,54 @@ python -m tox -e windows-dev # (Windows)
 
 This will
 
-1. Create a virtual environment located at `.tox/dev`.
-2. Install all the dependencies for PyBaMM, including the ones for documentation and development.
-3. Tell Python to use your local pybamm files when you use `import pybamm` anywhere on your system.
+a. Create a virtual environment located at `.tox/dev`.
+
+b. Install all the dependencies for PyBaMM, including the ones for documentation and development.
 
-Finally, activate your environment.
+c. Tell Python to use your local pybamm files when you use `import pybamm` anywhere on your system.
+
+5. Finally, activate your environment.
 
 
 ```bash
 source .tox/dev/bin/activate # (GNU/Linux and MacOS)
 #
-.tox\dev\Scripts\activate.bat # (Windows)
+.tox\windows-dev\Scripts\activate.bat # (Windows)
+```
+6. Inside the virtual environment, type:
+```bash
+pip install tox # (Skip this step if you used pipx above)
+```
+
+### 2. Without using tox
+1. Install virtual environment using the command below:
+```bash
+pip install virtualenv
+```
+2. Create a virtual environment `env` within your current directory using the command below:
+```bash
+python -m venv env 
```

Comment:
This will execute the built-in `venv` module. Do you mean `pip -m virtualenv` instead?

Reply:
> Thank you for this @Saransh-cpp , it's very useful to get feedback on the installation process.
> I'm not sure it's worth describing two cases (with tox - without tox) here, as it is already done in the documentation ([Install from source (developer install)](https://pybamm.readthedocs.io/en/latest/install/install-from-source.html)).
> Have you had a look at the documentation at

Thanks for the review! Should I remove the second step completely and use only tox?

Reply:
I actually took that step from the doc, now I think it would be better to just keep the tox installation in this file.

Reply:
Ah yes - the instructions for Windows in the docs mention `venv`. For the sake of consistency, these should updated as well to mention `virtualenv` instead of `venv`. 
If you have a moment to update the relevant page in the docs, that would be great, but don't worry if not.
Anyway, I think it would be best to only mention `virtualenv` here, adding that it must be install with `pip`, as you noted :)

Reply:
Yes I will update that. Should I replace ```python -m venv env``` with ```python -m virtualenv env```?

Reply:
Yep, exactly